WORKSPACE

```
@@ -87,16 +72,23 @@ mkl_repo(
     name = "mkl",
     root_env_var = "MKLROOT",
     urls = [
+        #when the issue with binutils will be solved, replace 2023.0 to 2024.2
```

`TODO` makes it more searchable.
```suggestion
        # TODO: when the issue with binutils will be solved, replace 2023.0 to 2024.2
```